T28444 : Test für QPS bauen

typ1804 opened the pull request.

CIO-Stefan committed:
T28444 : Test für QPS bauen

Schrank commented on src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:
```diff
@@ -90,6 +90,14 @@ public function saveAction()
 
+    public function loadAction()
+    {
+        $cron = new Mageone_Qps_Model_Cron();
```
Mage::getModel()

typ1804 replied:
done

Schrank commented on src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:
```diff
@@ -90,6 +90,14 @@ public function saveAction()
+    public function loadAction()
+    {
+        $cron = new Mageone_Qps_Model_Cron();
+        $cron->getRules();
```
Can you catch the exception here and if we are in admin context (not sure how to test) `Mage::getSession('admin/session')->addWarning(...)`

not 100% sure about the code, please check :D

typ1804 replied:
There is no exception to catch. It's already caught in the getRules-function

Schrank commented on src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:
```diff
@@ -12,7 +12,7 @@ public function indexAction(): void
     public function editAction()
     {
-        $id    = $this->getRequest()->getParam('id');
+        $id = $this->getRequest()->getParam('id');
```
be careful with these whitespace-only changes. normally no problem, but if customers want to review the changes, it is harder.

typ1804 replied:
This kind of white space shouldn't even happen. It’s just bad. All unnecessary white space is.

Schrank replied:
I'm happy to challenge that opinion. With a beer (or something else) on a nice evening :-)

CIO-Stefan committed:
T28444 : Test für QPS bauen

norgeindian committed:
Explain where to find the rules

I would suggest to explain in the README where to finde the rules to enable and disable them.

Schrank committed:
Merge pull request #7 from norgeindian/patch-1

Explain where to find the rules

Schrank committed:
Update Mageone_Qps.csv

Schrank committed:
Extend README.md

- Configure cronjob
- Run install script
- clean cache
- add username + key

phizab committed:
Fixed: undefined variable used

The $data variable isn't defined in method validateHttpVersion(), so I think $line variable was intended to be used here.

Schrank committed:
Merge pull request #9 from phizab/patch-1

Fixed: undefined variable used
